ignite/contrib/metrics/regression/median_absolute_percentage_error.py

```diff
@@ -31,6 +31,15 @@ class MedianAbsolutePercentageError(_BaseRegressionEpoch):
 
     __ https://arxiv.org/abs/1809.03006
 
+    Args:
```

Comment:
It misses `device` argument in `__init__`

Reply:
Actually, they all missing device here as contrib metrics were not yet "finished"

Reply:
some metrics have `device` argument, some didn't.
Should `device` be removed or left as it is ?

Reply:
IMO `device` should be removed. The docstring must be related to the code even it is not yet perfect.

Reply:
Fortunately, docstring and code matched perfectly. 😄